src/Raven.Server/Web/Studio/Sharding/Commands/GetConflictResolutionCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Commands;
+using Raven.Server.Documents.Sharding.Handlers;
+using Raven.Server.Web.Operations;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Studio.Sharding.Commands
+{
+    internal class GetConflictResolutionCommand : ShardedStreamCommand
```

Comment:
I don't think we need that if we would use the `AbstractHandlerProxyReadProcessor` instead

Reply:
done